pom.xml

```diff
@@ -127,6 +127,10 @@
           <groupId>com.fasterxml.jackson.core</groupId>
           <artifactId>jackson-databind</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
The `org.apache.ivy:ivy` transitive dependency is present in `org.apache.spark:spark-core_2.11` dependency of `io.cdap.cdap:cdap-spark-core2_2.11` module. Ideally the transitive dependency should be excluded from direct dependency in CDAP's cdap-spark-core2_2.11 module. ([link](https://github.com/cdapio/cdap/blob/develop/cdap-spark-core2_2.11/pom.xml#L92))
This would ensure that the vulnerability is fixed at all places.

Output of `mvn dependency:tree -Dincludes='org.apache.ivy:ivy'`:
```
[INFO] \- io.cdap.cdap:hydrator-test:jar:6.9.0-SNAPSHOT:provided
[INFO]    \- io.cdap.cdap:cdap-unit-test:jar:6.9.0-SNAPSHOT:provided
[INFO]       \- io.cdap.cdap:cdap-spark-core2_2.11:jar:6.9.0-SNAPSHOT:provided
[INFO]          \- org.apache.spark:spark-core_2.11:jar:2.1.3:provided
[INFO]             \- org.apache.ivy:ivy:jar:2.4.0:provided
[INFO] ------------------------------------------------------
```

Reply:
I think this is ok assuming we don't have any tests that use this functionality.